client/src/index.html

```diff
@@ -104,8 +104,8 @@ <h4><a class="title-content" ui-sref="index">{{ AppCtrl.project.name }}</a></h4>
 </div>
 
 <!-- scripts loaded in body to prevent page from 'hanging'-->
-<script src="js/vendor.min.js"></script>
-<script src="js/bhima.min.js"></script>
+<script src="vendor.min.js"></script>
+<script src="js/dist/bhima.min.js"></script>
```

Comment:
@mbayopanda thanks for the comment! I will update these to be clearer for the build process. 

The organisation of these files isn't as important as `client/src` but should still be considered.

The reason for the different folder is because of the 'cleanup' process. The last build process simply deleted the entire `bin/client` folder on the `client-build` and didn't delete anything on the `watch` method. This iteration tries to delete smaller folders only when they are used, for example `clean-js` will only delete the `bin/client/js` folder when a new `bhima.min.js` is being generated.  For this reason nothing else was but in the `bin/client/js` folder because it would have been deleted at the time. 

I propose to try and keep the targetted folder clean style but factor in your comments we make these files build into seperate js subfolder, for example: 

```
- bin/ 
- - bin/client 
- - - bin/client/js
- - - - bin/client/js/bhima/
- - - - - bhima.min.js
- - - - bin/client/js/vendor/
- - - - - vendor.min.js
```

The same goes for CSS as per your previous comment.